core/src/main/scala/codesearch/core/regex/lexer/Tokenizer.scala

```diff
@@ -10,15 +10,18 @@ object Tokenizer {
 
   private def endForCharSet[_: P] = P("]").!
 
+  // ** A POSIX character class, e.g. `[:alpha:]`. */
```

Comment:
Sorry, I messed this up a bit :(

```suggestion
  /** A POSIX character class, e.g. `[:alpha:]`. */
```